lib/opentelemetry/trace/batch_span_processor.lua

```diff
@@ -9,13 +16,35 @@ local mt = {
     __index = _M
 }
 
+local function report_dropped_spans(count, reason)
+    otel_global.metrics_reporter:add_to_counter(
+        dropped_spans_metric, count, { reason = reason })
+end
+
+local function report_result(success, err, batch_size)
+    if success then
+        otel_global.metrics_reporter:add_to_counter(
+            export_success_metric, 1)
+        otel_global.metrics_reporter:add_to_counter(
+            exported_spans_metric, batch_size)
+    else
+        err = err or "unknown"
+        otel_global.metrics_reporter:add_to_counter(
+            exporter_failure_metric, 1, { reason = err })
+        report_dropped_spans(batch_size, err)
+    end
+end
+
 local function process_batches(premature, self, batches)
     if premature then
         return
     end
+    otel_global.metrics_reporter:observe_value(buffer_utilization_metric, #self.queue / self.max_queue_size)
```

Comment:
This should be `self:get_queue_size() / self.max_queue_size`?

Reply:
You are correct. Due to how the data is processed, I updated to use `self:get_queue_size()` within `on_end`: a12aed1c5d396e566a745f6d31469ad1efebae34